### PR TITLE
Fix(DeskFilter): redesign by bug 1274

### PR DESCRIPTION
**Pull request by doronlnd84**
[1274](https://dev.azure.com/spectrumFactory/CoronaI/_sprints/backlog/CoronaI%20Team/CoronaI/Sprint%2013?workitem=1274)

### CODE REVIEW

File: client/src/components/App/Content/LandingPage/InvestigationTable/DeskFilter/DeskFilterStyles.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { makeStyles } from '@material-ui/styles';
 
 const useStyles = makeStyles((theme: Theme) => ({
     card: {
-        padding: theme.spacing(1),
+        // padding: theme.spacing(1),
```
Comment (roymaniv) [inline client/src/components/App/Content/LandingPage/InvestigationTable/DeskFilter/DeskFilterStyles.ts:6]: is the comment needed?

File: client/src/components/App/Content/LandingPage/InvestigationTable/DeskFilter/DeskFilterStyles.ts [hunk 2]
```diff
@@ -17,13 +17,13 @@ const useStyles = makeStyles((theme: Theme) => ({
         textOverflow: 'ellipsis',
     },
     autocomplete: {
-        marginTop: theme.spacing(1),
-        width: '80%',
+        // marginTop: theme.spacing(1),
```
Comment (roymaniv) [inline client/src/components/App/Content/LandingPage/InvestigationTable/DeskFilter/DeskFilterStyles.ts:20]: same here

File: client/src/components/App/Content/LandingPage/InvestigationTable/DeskFilter/DeskFilterStyles.ts [hunk 3]
```diff
@@ -17,13 +16,12 @@ const useStyles = makeStyles((theme: Theme) => ({
...
-        marginTop: theme.spacing(1),
-        width: '80%',
+        width: '70%',
         height: '60%',
-        marginBottom: theme.spacing(1),
+         marginBottom: theme.spacing(1),
 
         '& .MuiInputBase-root': {
-            fontSize: '0.7vw'
+            fontSize: '0.5vw'
```
Comment (y-golde) [inline client/src/components/App/Content/LandingPage/InvestigationTable/DeskFilter/DeskFilterStyles.ts:24]: why is the font size not the same as the card's font size(line 10)? 
this was my reasoning to write this line in the first place :0

**Review by y-golde — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/DeskFilter/DeskFilterStyles.ts (hunk 3 above)
Comment (doronlnd84) [reply client/src/components/App/Content/LandingPage/InvestigationTable/DeskFilter/DeskFilterStyles.ts]: My bad...

**Review by y-golde — APPROVED** [review]

**Review by roymaniv — APPROVED** [review]

**Pull request merged**